firmware/common/oversample.hpp

```diff
@@ -57,10 +57,9 @@
  * The oversample rate is used to increase the sample rate to improve SNR and quality.
  * This is also used as the interpolation rate when replaying captures. */
 inline OversampleRate get_oversample_rate(uint32_t sample_rate) {
-    if (sample_rate < 50'000) return OversampleRate::x128;  // 25k..12k5, prepared for future, OVS ok, but decim. x128 still not implemented.
-    if (sample_rate < 100'000) return OversampleRate::x64;  // 50k, prepared for future, OVS ok, but decim. x64 still not implemented.
-    if (sample_rate < 150'000) return OversampleRate::x32;  // 100k needs x32
-    if (sample_rate < 250'000) return OversampleRate::x16;  // 150k needs x16
+    if (sample_rate < 25'000) return OversampleRate::x64;   // 16k , 12k5.
```

Comment:
Looks like you changed 25K to be 32K in the SPEC list. Should you change the threshold on line 60?

Reply:
I'd like it if these values were kept parallel with the SPEC bandwidths. There are so many magic numbers already, it would help make it easier to see where these values are coming from.